family2family/src/main/java/org/cbtf/f2f/model/Category.java

```diff
@@ -25,12 +25,15 @@
 
     private Boolean active;
 
+    public Category(String description, Boolean active, List<Subcategory> subCategories) {
```

Comment:
http://localhost:8080/internal-api/Category errors. Complains about not having a default constructor.